### DISCUSSION TITLE
I have not received education benefits. 3 days ago approved.

### DISCUSSION THREAD

**Discussion by ApolloChenTw** [community/community#145664] [GitHub Education] [answered]
My request Submitted Nov 24, 2024

After seeing so many people have the same problem as me , I am just wondering how long I will be waiting is normal.

![image](https://github.com/user-attachments/assets/2e98868d-1ebc-4554-b94a-42bf6e8cd54b)

**Comment by davevad93** [comment, marked as answer]
Hi @ApolloChenTw ,

Don't worry it's normal, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ApolloChenTw was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by ApolloChenTw** [reply]
The status label turned purple!
Thanks to the community and @davevad93 

![image](https://github.com/user-attachments/assets/81c29848-fafb-4a42-b74b-06a476bf28e6)
